### DISCUSSION TITLE
Trigger + chamber door

### DISCUSSION THREAD

**Discussion by ItzAlmaZ** [BEEmod/BEE2-items#4132] [Tech Support / Q&A] [answered]
can the door open with an once trigger be closed (not chamberlock)

**Comment by slambo311** [comment, marked as answer]
Yes, just make sure that it starts open.

**Reply by ItzAlmaZ** [reply]
You did not understand. Can a door opened earlier by a once trigger be closed by anything?

**Reply by vrad-exe** [reply]
It can using some logic gates. Connect a trigger once and a NOT gate to the door, then connect another trigger to the NOT gate. When the first trigger is hit, the door will open; when the second is then hit, it will close.

**Reply by ItzAlmaZ** [reply]
I'll try it
